tests/test_question_templates/question_expected_outputs/instructor/q01_multiple-choice/q01_multiple-choice.md

```diff
@@ -104,30 +104,30 @@ myst:
       t: 6
       part1:
         ans1:
-          value: 42
+          value: <code>42</code>
```

Comment:
See comment left on `assemble_server_py` function - Maybe we should move the `re.sub` call elsewhere so it doesn't effect the calls to it. Should we maybe put it in the `write_server_py` function so it only changes writing the file?